Userland/Libraries/LibC/signal.cpp

```diff
@@ -143,7 +143,7 @@ int sigpending(sigset_t* set)
 }
 
 // Signal 0 (the null signal) and Signal 32 (SIGCANCEL) are deliberately set to null here.
-// They are not intended to be resolved by strsignal(), getsignalname() or getsignalbyname().
```

Comment:
how about we rename these to sig2str() and str2sig() if those are in posix and the current ones aren't?